packages/plugin-lighthouse/src/lib/lighthouse-plugin.ts

```diff
@@ -1,5 +1,7 @@
 import { defaultConfig } from 'lighthouse';
+import { join } from 'path';
 import { AuditOutputs, PluginConfig } from '@code-pushup/models';
+import { createFileWriteRunnerConfig } from '@code-pushup/utils';
```

Comment:
Lets move this into `models/test/fixtures/runner.config.mock.ts` please

Reply:
Unfortunately, this function cannot live in the `models` package as it would cause a circular dependency. Until a separate testing utils package is created, Matthew advised me to add it to `utils` and account for one separate usage by duplicating the function logic.